README.md

```diff
@@ -78,25 +54,56 @@ Do this once:
 mv src/third_party/libreofficekit{,.old}
 ```
 
-Linux/MacOs
+**Linux/macOS**
 ```bash
 # link src/third_party/libreofficekit to your local libreofficekit/libreoffice-core
 ln -s ../libreofficekit/libreoffice-core src/third_party/libreofficekit
 ```
 
-Windows
+**Windows**
 ```bash
 # note you must use an administrator command prompt, and the source and target are flipped
 mklink /D ".\src\third_party\libreofficekit" "..\libreofficekit\libreoffice-core"
 ```
 
+When you make a new local build of `libreofficekit`, make a new Electron build with `UPDATE_LOK=true`:
+```shell
+UPDATE_LOK=true scripts/e build
+```
+
+## Using a debug build in another project
+```shell
+# copy zipped runtime files to ~/another_project
```

Comment:
unless we have many other uses cases lined up, I'd suggest using the monorepo folder name for clarity

Reply:
Instructions are provided in the PR for the monorepo, this is an open source project.

Reply:
See: https://github.com/coparse-inc/app-monorepo/pull/1921